configs/components/windows_puppet.json

```diff
@@ -1,4 +1,4 @@
 {
   "url": "git://github.delivery.puppetlabs.net/puppetlabs-puppet_for_the_win.git",
-  "ref": "refs/tags/4.4.0.1"
+  "ref": "refs/tags/4.4.0"
```

Comment:
I think @Iristyle's comment is outdated. That should be 4.4.0.1 to pick up the change for https://tickets.puppetlabs.com/browse/PA-225 which is to be released in puppet-agent 1.4.0.